### ISSUE TITLE
[a11y] Horizontal `Stepper` doesnt indicate state in another way than style

### ISSUE THREAD

**Issue by JLou** [AxaFrance/design-system#1655]
# Accessibility verification of the component: `Stepper`

## Detected issues

It should be possible to dermine the state of a `Step` in the `Stepper` through a11y properties,
Today the only difference is the css applied.

The most probable outcome is to use [aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current)

A reference is also available here: https://www.w3.org/WAI/tutorials/forms/multi-page/
